### PR TITLE
[PLAY-681] Player doesn't show an object with a Stroke and without a Fill

### PR DESCRIPTION
**Pull request by mcm69**

### CODE REVIEW

File: src/import/animatron-importer.js [hunk 1]
```diff
@@ -387,7 +389,7 @@ Import.leaf = function(type, src, parent, anim) {
     var trg = new Element();
     var hasUrl = !!src[1];
     if (!hasUrl &&
-        (type === TYPE_IMAGE || TYPE_AUDIO || TYPE_VIDEO)) {
+        (type === TYPE_IMAGE || type === TYPE_AUDIO || type === TYPE_VIDEO)) {
```
Comment (mcm69) [inline src/import/animatron-importer.js]: ¯\\_(ツ)_/¯

Comment (skavish) [inline src/import/animatron-importer.js]: wow

**Comment by skavish** [comment]
so Path2D was not actually used, right? because I remember there was some speed improv (claimed)

**Comment by mcm69** [comment]
I'm not sure when the typo in path2d.js was introduced - the test results are true (there were no usep2d check in the test player dist). So yes, it wasn't used for some time

**Pull request merged**